tests/extensions/Wyam.Highlight.Tests/HighlightFixture.cs

```diff
@@ -208,17 +207,19 @@ public void HighlightFailsForMissingLanguage()
                 IDocument document = new TestDocument(input);
                 IExecutionContext context = new TestExecutionContext()
                 {
-                    JsEngineFunc = () => new TestJsEngine()
+                    JsEngineFunc = () => TestJsEngine.Create()
                 };
 
                 Highlight highlight = new Highlight();
 
                 // When, Then
-                Assert.Throws<AggregateException>(() =>
+                AggregateException ex = Assert.Throws<AggregateException>(() =>
                 {
                     List<IDocument> results = highlight.Execute(new[] { document }, context).ToList();
                     Assert.IsNull(results, "Should never get here due to exception");
                 });
+                Assert.AreEqual(1, ex.InnerExceptions.Count);
+                StringAssert.Contains("Unknown language: \"zorg\"", ex.InnerExceptions[0].Message);
```

Comment:
This test was only asserting that an AggregateException was thrown, but it wasn't actually checking the message of the assertion! I added some extra assertions to ensure the right exception is being caught.